topics/epigenetics/slides/introduction_atac.html

```diff
@@ -0,0 +1,121 @@
+---
+layout: introduction_slides
+logo: "GTN"
+video: true
+
+title: "ATAC-Seq data analysis"
+type: "introduction"
+questions:
+  - What is ATAC-Seq?
+  - What are the quality parameters to check for each dataset?
+  - How to analyse ATAC-Seq data?
+objectives:
+  - Understand ATAC-Seq
+  - Quality Parameters for ATAC-Seq
+  - Understand Peak calling for ATAC-Seq
+requirements:
+time_estimation: "30min"
+key_points:
+  - Run quality control on every sequencing dataset before any other analyses
+  - Choose QC parameters carefully
+  - Re-run FastQC to check the impact of the quality control
+contributors:
+  - heylf
+
+---
+
+# What is ATAC sequencing?
+
+---
+
+### Where my data comes from?
+
+![](../images/atac-seq/atac.jpg)
+
+<small>
+[*Buenrostro et al. 2013 Nat Methods*](https://doi.org/10.1002/0471142727.mb2129s109)
+</small>
+
+- Usage of hyperactive Tn5 transposase to insert sequencing adapters into open chromatin regions.
+- After adapter attachment the DNA is sheared by the Transposase itself.
+
+???
+
+It requires sequence-specific excision of a locus containing 19 base-pair inverted repeats.
```

Comment:
I am not sure I understand this. There is a sequence bias but the Tn5 can cut everywhere...

Reply:
Maybe I missunderstood this paper then: ATAC-seq: A Method for Assaying Chromatin Accessibility 
Genome-Wide, Jason Buenrostro [2016]. It states:  Tn5 is a prokaryotic transposase, which endogenously functions through the “cut and paste” mechanism, requiring sequence-specific excision of a locus containing 19 base-pair inverted repeats.

What does this sentence then mean?

Reply:
The original transposase Tn5 indeed only recognize the 19bp inverted repeat (https://onlinelibrary.wiley.com/doi/full/10.1046/j.1365-2958.2003.03382.x) but the hyperactive Tn5 is less restrictive.

Reply:
check